src/less/components/file/index.less

```diff
@@ -1,63 +0,0 @@
-@import (reference) 'components/box/index';
```

Comment:
@lalithkarikelli, please revert changes to this file and do not include in this PR.  We will remove it in PR #650.

Reply:
Changes have been done, please review.